pkg/vm/vm.go

```diff
@@ -961,6 +982,16 @@ func makeArrayOfFalses(n int) []StackItem {
 	return items
 }
 
+func checkKey(key *Element) {
```

Comment:
This is a waaay generic name for a function like this. I'd propose something like `validateMapKey`.